pkg/plugin/packetparser/_cprog/packetparser.c

```diff
@@ -5,39 +5,40 @@
 #include "bpf_helpers.h"
 #include "bpf_endian.h"
 #include "packetparser.h"
+#include "conntrack.c"
+#include "conntrack.h"
 #include "retina_filter.c"
 #include "dynamic.h"
 
 char __license[] SEC("license") = "Dual MIT/GPL";
 
-
-
 struct tcpmetadata {
 	__u32 seq; // TCP sequence number
 	__u32 ack_num; // TCP ack number
+	__u32 tsval; // TCP timestamp value
+	__u32 tsecr; // TCP timestamp echo reply
 	// TCP flags.
 	__u16 syn;
 	__u16 ack;
 	__u16 fin;
 	__u16 rst;
 	__u16 psh;
 	__u16 urg;
-	__u32 tsval; // TCP timestamp value
-	__u32 tsecr; // TCP timestamp echo reply
 };
 
 struct packet
 {
-	// 5 tuple.
+	__u64 ts; // timestamp in nanoseconds
```

Comment:
maybe rename to sth like `t_nsec` to disambiguate from the other timestamps that are 32-bit

Reply:
not a blocker, but you liked and resolved the comment without making any changes :D